packages/config/package.json

```diff
@@ -26,17 +26,14 @@
     "@truffle/provider": "^0.2.45",
     "conf": "^10.0.2",
     "find-up": "^2.1.0",
-    "lodash.assignin": "^4.2.0",
     "lodash.merge": "^4.6.2",
     "module": "^1.2.5",
     "original-require": "^1.0.1"
   },
   "devDependencies": {
     "@types/configstore": "^4.0.0",
     "@types/find-up": "^2.1.0",
-    "@types/lodash.assignin": "^4.2.6",
     "@types/lodash.merge": "^4.6.6",
-    "@types/lodash.pick": "^4.4.6",
```

Comment:
Oh hey this is unused? Might be worth submitting a PR to get rid of that separately from this...